packages/services/f-braze-adapter/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-braze-adapter",
   "description": "Fozzie Metadata Service",
-  "version": "5.0.1",
```

Comment:
Should be `v0.6.0`.

Reply:
Whoops. The typo was in the changelog. Will update it to be 5.0.1